pandas/core/generic.py

```diff
@@ -7825,6 +7828,49 @@ def _doc_parms(cls):
 
 """
 
+_any_also = """\
+See Also
+--------
+pandas.DataFrame.all : Return whether all elements are True \
```

Comment:
I don't think you need the trailing `\` here do you?

Reply:
to be clear, you do need the one on the first line, jsut not these.

Reply:
yes, and the same for the ones below as well

Reply:
We want them, else we get long lines in the text docstring liek

```
One dimensional boolean pandas.Series is returned. Unlike pandas.DataFrame.all, pandas.DataFrame.any performs OR operation; in other word, if any of the values along the specified axis is True, pandas.DataFrame.any will return True.

```

Reply:
@TomAugspurger & @jorisvandenbossche , let's say I removed `\` and , then the result of `git diff origin/master -u -- "*.py" | flake8 --diff` is going to be `pandas/core/generic.py:7834:80: E501 line too long (83 > 79 characters)`, since the line is `pandas.DataFrame.all : Return whether all elements are True over requested axis.` - I don't want to break the line with `\n`, instead, I'm using `\`. There is exactly same reason behind the cases I used `\`.